modules/sharedaddy/sharing-sources.php

```diff
@@ -76,7 +74,7 @@ public function get_link( $url, $text, $title, $query = '', $id = false ) {
 			( $id ? esc_attr( $id ) : '' ),
 			implode( ' ', $klasses ),
 			$url,
-			( $this->open_links == 'new' ) ? ' target="_blank"' : '',
+			( $this->open_link_in_new == true ) ? ' target="_blank"' : '',
```

Comment:
As much as I hate them, I believe our code style is to use yoda conditionals ( @georgestephanis can you confirm? )